mne/time_frequency/psd.py

```diff
@@ -113,15 +116,20 @@ def _check_nfft(n, n_fft, n_overlap):
 
 
 @verbose
-def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
-                       tmax=None, n_fft=256, n_overlap=0, proj=False,
-                       n_jobs=1, verbose=None):
+def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf,
+                       tmin=None, tmax=None, n_fft=256, n_overlap=0,
+                       proj=False, method='welch',  mt_bandwidth=None,
+                       mt_adaptive=False, mt_low_bias=True,
+                       mt_normalization='length', sfreq=None, n_jobs=1,
+                       verbose=None):
     """Compute power spectral density with average periodograms.
 
     Parameters
     ----------
-    epochs : instance of Epochs
-        The epochs.
+    epochs : instance of Epochs | array of shape(n_epochs, n_channels, n_times)
```

Comment:
re: the docstring....it was my OCD's fault. Adding the extra space means I have to put it on a second line :)

re: the array. How do you imagine a private function working? As in, create a function called "_prepare_psd_data" or something? Regarding the question of whether to do this in the first place, I tend to be partial towards letting people do things with arrays if they want to. This is partially out of my own preference, but partially because there are some functions that let you use arrays in MNE, some that don't. E.g., the cwt_morlet functions let you use arrays (I think they require it actually), but these compute_XXX_psd functions didn't, which was a bit confusing for me. So my tendency would be to add array functionality to keep things flexible.

This makes me wonder - perhaps it would be worth having a function to create a PSD that only allows arrays, and then have these "compute_XXX_psd" functions call that instead. So rather than baking in array functionality to these functions, there's a function called simply `psd` that would have the `method` flag and would work for an n-dimensional dataset as long as the final dimension is time. Then both `compute_epochs_psd` and `compute_raw_psd` would pull the data from epochs and pass it to the `psd` function. This could be similar to how the `tfr_XXX` functions work.